app/src/test/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -30,52 +29,31 @@ internal class PopularMoviesViewModelTest {
     )
```

Comment:
I don't think we need so many list objects for this case :D the most important thing here, is that the mapping was triggered, when use case is invoked.

I think in this case it is better to use mockks than real list, You can just use: 

val moviesModel: List<MovieModel> = mockk(relaxed = true)
val movies: List<Movie> = mockk() 

and mockk the toUi function with:

@ BeforeEach
    fun setup() {
        mockkStatic(List<MovieModel>::toUi)
    }
 
with usage of :

every { moviesModel.toUi() } returns movies

In the loaded state test

WARNING! 

Please be aware of, that using beforeEach can slow down the tests, as this is triggered before every test.
As an alternative, we can use @BeforeAll annotation for mockkStatic(..), but then, the all test class have to be annotated as PER_CLASS -> that would force us to declare view model every time for each test. So in this case i would go with beforeEach and mockkStatic in there approach

Reply:
I read this article that you send me [Link](https://www.arhohuttunen.com/junit-5-test-lifecycle/)

Reply:
Done